conf/messages

```diff
@@ -106,4 +111,16 @@ efrbsStartDate.error.required = The date that the employer-financed retirement b
 efrbsStartDate.error.invalid = Enter a real date that the employer-financed retirement benefits scheme started
 efrbsStartDate.error.future = The date that the employer-financed retirement benefits scheme started must be today or in the past
 efrbsStartDate.error.past = The date that the employer-financed retirement benefits scheme started must be after 31 December 1499
-efrbsStartDate.hint = For example, 31 3 1980
\ No newline at end of file
+efrbsStartDate.hint = For example, 31 3 1980
+
+setUpInAdditionToWillTrust.title = Is this is addition to a will trust?
+setUpInAdditionToWillTrust.heading = Is this is addition to a will trust?
+setUpInAdditionToWillTrust.checkYourAnswersLabel = Is this is addition to a will trust?
```

Comment:
Is this **in** addition to a will trust

Reply:
I would have definitely spotted that when I started manual testing it, but I'll update that now